tools/docker/builder/openwrt/Dockerfile

```diff
@@ -95,7 +95,15 @@ RUN cp feeds.conf.default feeds.conf \
     : "Installing intel feed doesn't correctly regenerate kernel .package-info"; \
     : "force regeneration by removing it"; \
     rm -rf tmp \
-    && make defconfig \
+    && make defconfig; \
+    : "We need to keep the hashes in the firmware, to later know whether an upgrade is needed:"; \
+    mkdir -p files/etc \
+    && printf '%s=%s\n' "OPENWRT_REPOSITORY" "$OPENWRT_REPOSITORY" >> files/etc/prplwrt-version \
+    && printf '%s=%s\n' "OPENWRT_VERSION" "$OPENWRT_VERSION" >> files/etc/prplwrt-version \
+    && printf '%s=%s\n' "PRPLMESH_VARIANT" "$PRPLMESH_VARIANT" >> files/etc/prplwrt-version \
+    && printf '%s=%s\n' "PRPL_FEED" "$PRPL_FEED" >> files/etc/prplwrt-version \
+    && printf '%s=%s\n' "INTEL_FEED" "$INTEL_FEED" >> files/etc/prplwrt-version \
+    && printf '%s=%s\n' "IWLWAV_FEED" "$IWLWAV_FEED" >> files/etc/prplwrt-version \
```

Comment:
I like the suggestion, but the upcoming prplwrt release will use around 9 feeds for the rax40, so we will need to store them somewhere else anyway (unless we store them all in BUG_URL? :eyes: ).

So for now and also because I need this for the current task I'm working on, I'll keep them in  the prplwrt-release file.

Reply:
(nitpick): Any reason to prefer printf here over echo?
`echo "OPENWRT_REPOSITORY=$OPENWRT_REPOSITORY" >> files/etc/prplwrt-version` seems simpler to me :-)